Editor/Window/Components/StatusIndicator.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace AmazonGameLift.Editor
+{
+    public class StatusIndicator : VisualElement
+    {
+        public new class UxmlFactory : UxmlFactory<StatusIndicator> { }
+
+        private Label _text => this.Q<Label>();
```

Comment:
Please rename to `_label` just for clarity

Reply:
renamed